tee-worker/core-primitives/stf-executor/src/executor.rs

```diff
@@ -136,14 +139,17 @@ where
 		) {
 			Err(e) => {
 				error!("Stf execute failed: {:?}", e);
-				return Ok(ExecutedOperation::failed(top_or_hash, extrinsic_call_backs))
+				let rpc_response_value: Vec<u8> = trusted_operation.req_hash().map(|h| {
```

Comment:
@Kailai-Wang I think we can do the same to successful TOPs. TOPs can return result value that implements scalec's `Encode` and in case `req_has` is present encode it here and put it in `ExecutedOperation`. What do you think ?

This could lead us to removal of `https://github.com/litentry/litentry-parachain/blob/055b19d358089b0ed2f9571a23badbd82c638f6d/tee-worker/app-libs/stf/src/trusted_call.rs#L280`

Reply:
Hmm I'm not sure if I get you. You can do this for `ErrorResponse` because the failed TOP has the error of the same type. The success TOP would have different Response types even though they are all encodable. How would you write the `Ok(res)` here? if you mean `res` is already encoded, I don't see much difference from the current impl..

Reply:
They can be wrapped into enum (just like `StfError`) and handled in the same way.

Reply:
@Kailai-Wang please check this [commit](https://github.com/litentry/litentry-parachain/commit/84de0ee5af337a9039c299828e719dd3e0eef66e)